src/ontology/mondo-edit.obo

```diff
@@ -534940,7 +534941,7 @@ xref: OMIM:619058 {source="MONDO:equivalentTo"}
 xref: UMLS:C5436710 {source="MEDGEN:1763505", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
 is_a: MONDO:0033885 {source="OMIM:619058"} ! mitochondrial complex IV deficiency, nuclear-type
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/24990 {source="OMIM:619058"}
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/24990
```

Comment:
The omim source was removed because this is a provisional gene to phenotpye/disease relationship https://omim.org/entry/619058

Reply:
This removal is correct based on our rules for automation; however, based on the text in OMIM, and the definition for this term, I added it back.